tests/integration/fixtures/helm.py

```diff
@@ -147,6 +158,17 @@ async def matrix_stack(
         ]
     values["matrixTools"].setdefault("image", {})
     values["matrixTools"]["image"] = loaded_matrix_tools
+    values["matrixRTC"]["hostAliases"] = [
+        {
+            "ip": ingress,
+            "hostnames": [
+                generated_data.server_name,
+                f"synapse.{generated_data.server_name}",
+                f"mas.{generated_data.server_name}",
```

Comment:
Why is the MAS hostname required for matrixRTC?

Reply:
It's a remain from my tests, should be dropped